Symphony/Classes/LPRegion.h

```diff
@@ -13,8 +13,8 @@ NS_ASSUME_NONNULL_BEGIN
 @interface LPRegion : NSObject
 
 @property (nonatomic, strong) NSString *name;
-@property (nonatomic, strong) NSString *lon;
-@property (nonatomic, strong) NSString *lat;
+@property (nonatomic, strong) float lon;
```

Comment:
Do confirm that a float has the requisite accuracy for capturing lat/long

Reply:
android lat long uses float, but you are right i think double will be better, changed